CHANGELOG.md

```diff
@@ -23,7 +23,9 @@ streamlined to use Sphinx from now on.
  - use C++11 `using` instead of `typedef`
  - removed `Config` suffix in file names #1965
  - `gasConfig` is now `density`
- - `speciesDefinition`: simplified `Particles<>` interface #1711 #1942
+ - `speciesDefinition`:
+   - simplified `Particles<>` interface #1711 #1942
+   - `ionizer< ... >` became a sequence of `ionizers< ... >` #1999
```

Comment:
offline: it's a feature we want included in 0.3.0 and we had to branch early this time. usually, we don't add features to already open release candidates, only bug fixes